src/support.html

```diff
@@ -99,7 +99,9 @@
         is present in your Zigbee2MQTT configuration:
         <pre><code>serial:
           adapter: ezsp
-          port: /dev/ttyAMA1</code></pre>
+          port: /dev/ttyAMA1
+          baudrate: 115200
+          rtscts: false</code></pre>
```

Comment:
It should be set, as we have RTS/CTS connected and the signals enabled in the radio firmware

```suggestion
          rtscts: true</code></pre>
```